modules/githubIssueParser.js

```diff
@@ -1,120 +1,107 @@
-										}
-										else if (data.pull_request.url && data.draft) {
-											status = client.emojis.cache.find(emoji => emoji.name === 'pr_draft');
-										}
-										else if (data.pull_request.url && data.state === 'open') {
-											status = client.emojis.cache.find(emoji => emoji.name === 'pr_opened');
-										}
-										else if (data.pull_request.url && data.pull_request.merged_at) {
-											status = client.emojis.cache.find(emoji => emoji.name === 'pr_merged');
-										}
-										else if (data.pull_request.url && data.state === 'closed') {
-											status = client.emojis.cache.find(emoji => emoji.name === 'pr_closed');
-										}
-										else {
-											status = client.emojis.cache.find(emoji => emoji.name === 'spoopy');
-										}
-										// Push new pretty link to output array
-										output.push(`[${status.toString()} ${tag}, @${data.user.login}: ${data.title}](https://github.com/LMMS/lmms/issues/${tag.substring(1)})`);
-									}
-								}
-								if (response.status === 422 || response.status === 403) {
-									const errorEmbed = new EmbedBuilder()
-										.setColor(red)
-										.setDescription('Slow down. The GitHub API is displeased.');
-									// Send it
-									return await message.channel.send({ embeds: [errorEmbed] });
-								}
-							})
-							.catch((err) => {
-								console.log(err);
-							});
-
-						if (output.length > 9) {
-							break;
-						}
-					}
-					if (output.length) {
-						// Build output embed
-						const parseEmbed = new EmbedBuilder()
-							.setColor(green)
-							.setDescription(output.join('\n'));
-						// Send it
-						return await message.channel.send({ embeds: [parseEmbed] });
-					}
-				}
-			}
-
-		});
-	},
+    async execute(client) {
+
+        client.on(Events.MessageCreate, async message => {
+            if (message.author.bot) return;
+
+            // Only process messages containing a #
+            if (!message.content.includes('#')) return;
+
+            // Remove code blocks and inline code
+            const noBlockCode = message.content.replace(/```[\s\S]*?```/g, '');
+            const noInlineCode = noBlockCode.replace(/`[^`]*`/g, '');
+
+            // Match patterns:
+            // 1. #123
+            // 2. repo#123
+            // 3. org/repo#123
+            const regex = /(?:(?<org>[A-Za-z0-9_.-]+)\/(?<repo>[A-Za-z0-9_.-]+)|(?<repoOnly>[A-Za-z0-9_.-]+))?#(?<issue>\d{3,5})(?!\d)/g;
+
+            const matches = [...noInlineCode.matchAll(regex)];
+
+            if (!matches.length) return;
+
+            const output = [];
+            const seen = new Set();
+
+            for (const m of matches) {
+                const org = m.groups.org || github.allowedOrgs[0];
+                const repo = m.groups.repo || m.groups.repoOnly || github.defaultRepo;
+                const issue = parseInt(m.groups.issue);
+
+                if (!github.allowedOrgs.includes(org)) continue;
```

Comment:
This should make enough leeway for people to not mind about capitalization. GitHub accounts are case-insensitive anyways
```suggestion
                if (!github.allowedOrgs.map((str) => str.toLowerCase()).includes(org.toLowerCase())) continue;
```